xtask/src/runchecks.rs

```diff
@@ -249,6 +249,12 @@ fn burn_dataset_features_std() {
     cargo_doc(["-p", "burn-dataset", "--all-features"].into());
 }
 
+// Test burn-candle with accelerate (macOS)
+#[cfg(target_os = "macos")]
```

Comment:
Just one question: Can not run on `Linux`?

Reply:
Nope. This is specifically leveraging the [Accelerate](https://developer.apple.com/documentation/accelerate) framework on macOS.

Reply:
Thank you! Can you add a comment with a link in `runchecks.rs`? In this way we do not forget why there is that `cfg`.